docs/manual/_sources/index.rst.txt

```diff
@@ -23,10 +24,12 @@ This manual refers to the following versions of pytket and extensions:
     manual_assertion.rst
 
 .. toctree::
-	:caption: More documentation:
-	:maxdepth: 1
+    :caption: More Documentation:
```

Comment:
Is this working fine with the removed space? Have you checked that?

Reply:
Yes, I double checked that it builds correctly. Thanks.